topics/go/algorithms/strings/reverse/reverse.go

```diff
@@ -2,34 +2,29 @@ package strings
 
 func ReverseString(str string) string {
 
-	// Here we create a slice of codePoints.
-	codePoints := []rune(str)
+	// Here we create a slice of r.
+	r := []rune(str)
```

Comment:
This would have to be `runes` now since this is a collection.  `r` for a single rune but `runes` for the collection.